SECURITY.md

```diff
@@ -0,0 +1,50 @@
+# Security Policy
+This document outlines security policy and procedures for the CrowdStrike `falconpy` project.
++ [Supported Python versions](#supported-python-versions)
++ [Supported FalconPy versions](#supported-falconpy-versions)
++ [Reporting a potential security vulnerability](#reporting-a-potential-security-vulnerability)
++ [Disclosure and Mitigation Process](#disclosure-and-mitigation-process)
+
+## Supported Python versions
+
+FalconPy functionality is only tested to run under the following versions of Python.
+
+| Version | Supported |
+| :------- | :--------- |
+| 3.9.x   | :white_check_mark: |
+| 3.8.x   | :white_check_mark: |
+| 3.7.x   | :white_check_mark: |
+| 3.6.x   | :white_check_mark: |
+| <= 3.5  | :x: |
+| <= 2.x.x | :x: |
+
+## Supported FalconPy versions
+
+We release patches for security vulnerabilities as they are discovered. 
+
+Version eligibility for receiving patches depends on the CVSS v3.0 Rating:
+
+| CVSS v3.0 | Supported Versions |
+| :------- | :-------------------- |
+| 9.0 - 10.0 | Releases within the past three months |
+| 4.0 - 8.9 | Most recent release |
+
+## Reporting a potential security vulnerability
+
+Please report suspected security vulnerabilities to __falconpy@crowdstrike.com__. 
```

Comment:
Does it make sense to bound upstream vs direct emails? For example people could open tickets upstream for lower severity issues, which the community (as it builds over time) might be able to help fix. We could use the direct email for issues like remote code execution (or some other 0day scenario that warrants a high CVSS score).

Reply:
I like the idea of community involvement for fixes over time.  I worry that asking them to determine which route to follow based upon CVSS score will get missed.  

These reports should also be accepted via a Bug, Discussion or PR (or to your comment below, via the BB program).  I will flush the text out to describe the other avenues.